Cargo.toml

```diff
@@ -50,6 +50,7 @@ static_assertions = "1.1.0"
 strum = "0.25"
 strum_macros = "0.25"
 sysinfo = "0.29"
+chrono = "*"
```

Comment:
I'm being pedantic, sorry, but the dependencies list is sorted in alphabetical order to make it easier to scan dependencies. Would you mind moving `chrono` above to make it alphabetical?

Reply:
No worries. I'm probably going to cleanup the code before merging, and might end up removing `chromo` altogether. But in the final version, if I leave it, I'll move it up to keep the alphabetical order.

Reply:
If you just need chrono for timestamps and don't actually care about the date/time then you can just use UNIX time (in milliseconds) as the prefix to the print statement